resources/analysisTools/snvPipeline/filter_PEoverlap.py

```diff
@@ -127,6 +132,7 @@ def performAnalysis(args):
     reference_file = pysam.Fastafile(args.refFileName)
 
     mode = "r"
+    multiple_iterators = False
     if args.alignmentFile[-4:] == ".bam":

```

Comment:
I am wondering about this code. What is 4 and 5? Is there some hidden assumption about the format of the filename? A comment would enormeously improve the readability, here.

Reply:
Updated the file extension detection here.